DMTN-248.tex

```diff
@@ -46,8 +46,245 @@
+\end{enumerate}
+
+These drawbacks could impose a risk to the particular science goals of brokers using full-sized or non-identical alerts.
+
+\textbf{Recommendation}: None at this time, this is primarily a technical question with little science impact.
+
+
+\section{Pre-filtered streams}\label{sec:prefilter}
+
+\textbf{Proposal}: Allow brokers to request a pre-filtered alert stream, for example to only include alerts in a certain sky 
+region or which meet other criteria (e.g., brightness limits, number of past detections).
+
+As with the multiple packet formats, the main motivation here is not scientific, but to help reduce brokers' processing costs 
+spent recieving unwanted alerts, or to enable Rubin Observatory to distribute alerts to more brokers.
+
+For example, due to the exponential relationship between the number of variable stars and their variability amplitude, and 
+also that of volume and distance modulus, an apparent magnitude limit that is 1 mag brighter than the nominal 5-sigma \texttt{DiaSource} 
+detection limit could reduce the alert stream data rate by 50\%.
+
+However, all seven full-stream brokers have committed to recieving the full stream, and the term downstream broker refers to 
+brokers which will ingest a filtered stream of alerts from the full-stream brokers.
+
+\textbf{Recommendation}: There is no need for Rubin Observatory to provide pre-filtered streams to brokers.
+
+
+\section{When and how to distribute delayed alerts}\label{sec:delayed}
+
+The Data Management System (DMS) is required to support the distribution of at least 40,000 alerts per single standard visit, 
+and for visits producing $\leq$40,000 alerts, no more than 1\% of them fail to have at least 98\% of its alerts distributed 
+within 60 seconds of image readout (based on LSR-REQ-0101 in \citeds{LSE-29}; OSS-REQ-0193 in \citeds{LSE-30}; 
+and DMS-REQ-0392 and -0393 in \citeds{LSE-61}).
+
+Furthermore, alert distribution should degrade gracefully beyond that limit, meaning that visits resulting in an excess of 
+40,000 of alerts should not cause any downtime for the Data Management System (DMS; \citeds{LSE-30}, \citeds{LSE-61}).
+It is also a requirement that all alerts be stored in an archival database and be available for retrieval 
+(OSS-REQ-0185 in \citeds{LSE-30}).
+
+This leaves the open question of what, from a science perspective, is the optimal way of dealing with delayed alerts.
+(Aspects of the technical implementation of a graceful degradation, such as distributing delayed alerts and alert archive 
+storage access, are out of scope for this document).
+
+There are three main options:
+
+\begin{enumerate}
+\item \textbf{Next-opportunity distribution via the alert stream}.
+Distribute delayed alerts as soon as possible.
+There are plenty of science goals that do not absolutely require alert distribution in 1 minute, and so distributing delayed 
```

Comment:
Maybe worth mentioning an referencing here that we have no science use cases requiring distribution in < 5 mins at this time (Broker workshop)

Reply:
I think we don't need to mention that here, so skipping (hard to reference a lack of something anyway).